bot0.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+func main() {
+	client, err := ethclient.Dial("https://mainnet.infura.io/v3/" + GetAPIKey("INFURA_KEY"))
+	if err != nil {
+		log.Fatal(err)
+	}
+	fmt.Println("we have a connection")
+
+	address := common.HexToAddress("0x3356c9a8f40f8e9c1d192a4347a76d18243fabc5")
```

Comment:
could you create a constant for this address?

Reply:
Done

Reply:
I don't see it

Reply:
Sorry for not pushing it, I just did